pkg/controller/rest/didclient/models.go

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
will these models ever be used in this repo?

Reply:
Yes, they are part of the spec.